packages/eslint-plugin-orbit-components/README.md

```diff
@@ -1,5 +1,7 @@
 # @kiwicom/eslint-plugin-orbit-components
 
+> **⚠️ This package is deprecated and will no longer be maintained.**
```

Comment:
What do you think about this formatting? [source](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)

> [!WARNING] 
> This package is deprecated and will no longer be maintained.

Reply:
Looks great tbh, I wasn't aware of that. But I don't know how that will look like on [NPM](https://www.npmjs.com/package/@kiwicom/eslint-plugin-orbit-components). They do refer to the Github markdown docs, so I can imagine it will work good 🤔